src/main/java/mate/academy/util/ConnectionUtil.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+import mate.academy.model.exception.DriverLoadingException;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/books";
+    private static final Properties DB_PROPERTIES;
+    private static final String DRIVER_LOADING_MESSAGE = "Can't load JDBC Driver";
+
+    static {
+        DB_PROPERTIES = new Properties();
+        DB_PROPERTIES.put("user","root");
+        DB_PROPERTIES.put("password", "956874lv");
```

Comment:
Ideally you should not commit your credentials to git. Instead, you can fetch them from system variables

Reply:
For now use System.getProperty. On my machine, I settled them in method. If I understand right, potential colleagues can set them manually or have a special file for credentials, which you fetched and fulfill with your creds, and the program gets values from it and this file content won't be pushed on git. Or something like that?

Reply:
Yea, that point was intended for your habit. For local dev, usually you will set them in the local config file. In prod/dev env they will be fetched form some specific services/vaults. But it is never a good idea to hardcode them